### ISSUE TITLE
Suntrust App issues

### ISSUE THREAD

**Issue by EnorMOZ** [n00neimp0rtant/xCon-Issues#599]
After installing beta2 touch id and other options stop showing up. Reverting back to beta1 bring back those options.  Let me know if you need anything to help with this.

**Comment by sheltop225** [comment]
Yep, 42-beta1 works fine for me also...beta2 doesn't.  I have 40-beta8 installed bc I need my dropbox working more than I need the touchID on my suntrust app though.
